odrivemotor/src/odriveCAN/odriveCAN.py

```diff
@@ -127,38 +127,34 @@ def reconfigure(self, config: ComponentConfig, dependencies: Mapping[ResourceNam
             self.set_node_id(new_nodeID)
 
     async def set_power(self, power: float, extra: Optional[Dict[str, Any]] = None, **kwargs):
+        if abs(power) < 0.1:
```

Comment:
let's go a little lower here since some brushless motors can move at lower powers than brushed because of less friction.